green-eggs-app/components/EggsUserProvider.jsx

```diff
@@ -0,0 +1,32 @@
+import { Text, StyleSheet, View } from 'react-native';
+import React, { createContext, useContext, useEffect, useState } from 'react';
+
+export const EggsUserContext = createContext(null);
+
+export const useEggsUserContext = () => {
+  return useContext(EggsUserContext);
+};
+
+export default function EggsUserProvider({ children }) {
+  const [currentEgg, setCurrentEgg] = useState(null);
```

Comment:
I think we have a variable already that's called currentEggs... could we maybe do a different name for this to avoid confusion? Or could you suggest a different name for the other one?

Reply:
Where is currentEggs and what's it for? I can't find it.